Issue 4507 - FSE. IB work fix on pattern inserting/block duplication/copy-paste

Fybex opened the pull request:
# Description

<!---
Please include a summary of the changes and the related issue.
Please also include relevant motivation and context.
--->

Fixes #4507 

# How Has This Been Tested?
Added patterns, checked if after adding a few similar patterns, IB have expected targets, triggers (#4320, but on FSE)

<!---
Please describe the tests that you ran to verify your changes.
Provide instructions so we can reproduce.
Please also list any relevant details for your test configuration.
--->

# Test checklist

<!--- Please remove the unnecessary checkbox --->

**_ Front/Back Testing _**

-   [ ] Test adding pattern in FSE
-   [ ] Test pattern IB in FSE
-   [ ] Test pattern IB after duplicating block
-   [ ] Check that the settings work on frontend

**_ Pre-Code Testing _**

-   [ ] Test adding pattern in FSE
-   [ ] Test pattern IB in FSE
-   [ ] Test pattern IB after duplicating block
-   [ ] Check that the settings work on frontend
-   [ ] Check no commented code and no unnecessary imports
-   [ ] Standards of the project have been followed
-   [ ] No errors/warnings on console

# Checklist

-   [X] My code follows the style guidelines of this project
-   [X] I have performed a self-review of my code
-   [X] I have commented my code, particularly in hard-to-understand areas
-   [X] My changes generate no new warnings/errors

Naaaaiix committed:
Remove lazy load
Improved withSuspense lifecycle

Fybex committed:
`getLastChangedBlock`: fix for FSE

Fybex committed:
`entityRecordsWrapper`: fix for FSE `addClearEntity`

Fybex committed:
Small fixes

Fybex committed:
Update package-lock.json

Naaaaiix committed:
Fix tests

Naaaaiix committed:
Remove console.log

Olekrut reviewed (changes requested):
https://jam.dev/c/5dc316b5-a9a9-40ec-8290-470f3ac671ac - backend
backend after refresh editor, but can't save
![image](https://user-images.githubusercontent.com/88893350/224650223-b3c6bfe9-21de-41d6-b412-1500dbd37953.png)


Frontend - 
![image](https://user-images.githubusercontent.com/88893350/224650132-004456f0-7a11-4ca1-9378-55ae1e132b4c.png)

Naaaaiix committed:
Implement lighter Suspense

Naaaaiix committed:
Fixes `withMaxiSuspense` just for the page load

Naaaaiix committed:
Merge branch 'master' into lighter-suspense

Naaaaiix committed:
Fix package-lock.json

Naaaaiix committed:
Replace the name of the loader HOC

Naaaaiix committed:
Merge branch 'master' of https://github.com/maxi-blocks/maxi-blocks

Naaaaiix committed:
Main Maxi block component optimization (#4558)

* Revert Suspense implementation

* Abstract RowBlockTemplate component
Reduce withMaxiProps load
Reduce MaxiBlockComponent load
Abstracted some MaxiBlockComponent methods
Avoid rendering Inspector if it's not selected

* Clean Text Maxi main file

* Clean code

* Library insert optimization (#4559)

* Refactored `onRequestInsertPattern`
Modified Library flow when clicking insert

* Fix package-lock.json

* Fix CORS
Clean code

* Fixes placeholder source

Naaaaiix committed:
Merge branch 'master' of https://github.com/maxi-blocks/maxi-blocks

Naaaaiix committed:
Merge branch 'master' into lighter-suspense

Naaaaiix committed:
Fix package.json

Naaaaiix committed:
Fix typo

Naaaaiix committed:
Fix tests

Fybex committed:
Merge branch 'master' into issue-4507

Fybex committed:
Merge branch 'lighter-suspense' into issue-4507

Naaaaiix committed:
Fix tests

Fybex committed:
Merge maxiBlockComponent

Naaaaiix committed:
Fixes compatibility with React 18

Fybex committed:
Merge branch 'lighter-suspense' into issue-4507

Fybex committed:
Fix error in FSE

Fybex committed:
Calculate correct `keepStylesOnEditor`

Fybex committed:
Remove unused imports, clgs

Fybex committed:
Merge branch 'master' into issue-4507

Fybex committed:
Remove commented import

Fybex committed:
Fix FSE preview

Fybex committed:
`getClientIdFromUniqueId` fix for template part

block which is in template(not part)

Fybex committed:
`getLastChangedBlocks `:use undoEdit.name for kind

Fybex committed:
Merge branch 'master' into issue-4507
